py/torch_tensorrt/dynamo/conversion/impl/select.py

```diff
@@ -370,3 +372,108 @@ def index(
             reshape_output = reshape_layer.get_output(0)
 
     return reshape_output
+
+
+def scatter_value(
+    ctx: ConversionContext,
+    target: Target,
+    source_ir: Optional[SourceIR],
+    name: str,
+    input: TRTTensor,
+    dim: int,
+    index: Union[TRTTensor, np.ndarray, torch.Tensor],
+    value: float,
+) -> TRTTensor:
+    if not isinstance(input, TRTTensor):
+        raise RuntimeError(
+            f"scatter_tensor received input {input} that is not part "
+            "of the TensorRT region!"
+        )
+    input_shape = input.shape
+    index_shape = index.shape
+    index_shape_list = list(index.shape)
+    if not (isinstance(index, TRTTensor)):
+        index = get_trt_tensor(ctx, index, f"_index_tensor")
+    if len(input_shape) != len(index_shape):
+        raise RuntimeError(f"The no of dimensions of input and index should be equal")
+    dim = get_positive_dim(dim, len(input_shape))
+    dynamic_shape = has_dynamic_shape(input.shape)
+    if dynamic_shape:
+        # Check whether slice target dim is dynamic shape dim
+        assert input.shape[dim] != -1, "Can't scatter on negative shape dimension!"
+
+    input_dims = len(input_shape)
+    for i in range(0, input_dims):
+        if i != dim and (index_shape[i] >= input.shape[i]):
+            raise RuntimeError(
+                f"cannot have index size greater than the input size along dimension {dim}"
+            )
```

Comment:
Does this cause an error in Torch? If so, it can be omitted, otherwise, it should go in the validator